From cb2699a15231f429b301fffb4278a092dc9c0a85 Mon Sep 17 00:00:00 2001
From: Willy Tarreau <w@1wt.eu>
Date: Thu, 24 Jan 2013 16:25:38 +0100
Subject: CLEANUP: config: maxcompcpuusage is never negative

No need to check for a negative value in the "maxcompcpuusage" argument,
it's an unsigned int.

Reported-by: Dinko Korunic <dkorunic@reflected.net>
---
 src/cfgparse.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/cfgparse.c b/src/cfgparse.c
index 5b8fc06..175e30c 100644
--- a/src/cfgparse.c
+++ b/src/cfgparse.c
@@ -902,7 +902,7 @@ int cfg_parse_global(const char *file, int linenum, char **args, int kwm)
 			goto out;
 		}
 		compress_min_idle = 100 - atoi(args[1]);
-		if (compress_min_idle < 0 || compress_min_idle > 100) {
+		if (compress_min_idle > 100) {
 			Alert("parsing [%s:%d] : '%s' expects an integer argument between 0 and 100.\n", file, linenum, args[0]);
 			err_code |= ERR_ALERT | ERR_FATAL;
 			goto out;
-- 
1.7.1

